packages/code-du-travail-frontend/src/conventions/ConventionExplainer.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+import { Section, Wrapper } from "@cdt/ui";
+import styled from "styled-components";
+
+const storedExpandedKey = "isConventionExplainerExpanded";
+
+const ConventionExplainer = () => {
+  const [expanded, setExpanded] = useState(true);
+
+  useEffect(() => {
+    const expanded = sessionStorage.getItem(storedExpandedKey) == "true";
```

Comment:
why not use `JSON.parse()` / `JSON.stringify(!expanded)` ?

Reply:
I don't see how that would be better @lionelB ? my arguments: 
- I don't see this value getting more complex than it is now (a boolean)
- I think it's easier to read as-is
- I guess it's more performant as-is

Reply:
because it's nitpick, can you just add a comment about the type coercion

> I think it's easier to read as-is

Not to me :) When I read the code I was wondering, "hey, why does it compare a string, weird ! This should be a Boolean" and then I remember that storage are string based.

Reply:
actually I removed completely this localStorage based behaviour because I switched to using the `Accordion` component which doesn't support it